debian/patches/Update-LuaTeX-support

```diff
@@ -0,0 +1,84 @@
+From: YOKOTA Hiroshi <qykth-git@users.noreply.github.com>
+Subject: [PATCH] Update LuaTeX support
+
+* Remove unexpected floating point number to support Lua 5.3+ based LuaTeX
+* Update LuaTeX hack
+
+---
+ doc/texinfo.tex |  8 ++++----
+ doc/txi-ja.tex  | 28 ----------------------------
+ 2 files changed, 4 insertions(+), 32 deletions(-)
+
+Index: b/doc/texinfo.tex
+===================================================================
+--- a/doc/texinfo.tex
++++ b/doc/texinfo.tex
+@@ -3,7 +3,7 @@
+ % Load plain if necessary, i.e., if running under initex.
+ \expandafter\ifx\csname fmtname\endcsname\relax\input plain\fi
+ %
+-\def\texinfoversion{2019-09-08.12}
++\def\texinfoversion{2019-09-20.22}
+ %
+ % Copyright 1985, 1986, 1988, 1990-2019 Free Software Foundation, Inc.
+ %
+@@ -1052,7 +1052,7 @@ where each line of input produces a line
+             tex.sprint(
+               string.format(string.char(0x5c) .. string.char(0x25) .. '03o' ..
+                             string.char(0x5c) .. string.char(0x25) .. '03o',
+-                            (c / 256), (c % 256)))
++                            math.floor(c / 256), math.floor(c % 256)))
+           else
+             c = c - 0x10000
+             local c_hi = c / 1024 + 0xd800
+@@ -1062,8 +1062,8 @@ where each line of input produces a line
+                             string.char(0x5c) .. string.char(0x25) .. '03o' ..
+                             string.char(0x5c) .. string.char(0x25) .. '03o' ..
+                             string.char(0x5c) .. string.char(0x25) .. '03o',
+-                            (c_hi / 256), (c_hi % 256),
+-                            (c_lo / 256), (c_lo % 256)))
++                            math.floor(c_hi / 256), math.floor(c_hi % 256),
++                            math.floor(c_lo / 256), math.floor(c_lo % 256)))
+           end
+         end
+       end
```

Comment:
Are these upcoming changes? I just checked the texinfo git repo and don't see them. Otherwise they look fine to me.

Reply:
No.
This is my own patch and not send to upstream yet.
I think this is the time to send my patches to upstream.

Reply:
Update my [texinfo-ja](https://github.com/qykth-git/texinfo/tree/texinfo-ja) branch to describe each fix.

Reply:
Thanks, I will merge this, but please try to bring your changes upstream, that would help also other distributions.

Thanks a lot!